packages/core/src/components/cv-dropdown/cv-dropdown.vue

```diff
@@ -173,6 +173,9 @@ export default {
     document.body.removeEventListener('click', this.checkSlots);
     this.checkSlots();
   },
+  destroyed() {
```

Comment:
The library tends to use beforeDestroy as the Vue instance has been destroyed at this point.

Is there a good reason for it being in the destroyed hook?

Reply:
Thanks for the heads-up.

No special reason, I made the change and will test as soon as it is available.